### ISSUE TITLE
Remote Workspace Temp Refresh?

### ISSUE THREAD

**Issue by dwengs** [humy2833/FTP-Simple#40]
I was using FTP Simple and wanted to check if FTP Simple successfully adds or delete folders, files etc. To check this, I open Filezilla and I saw that FTP Simple works well.
But, "remote directory open to workspace" option, seems not updating folders or files. So, I closed VSCode and connected FTP and remote workspace, but no luck. There was old files that i deleted minutes ago.

"Find in finder" (or explorer maybe in windows) shows TMP directory. I didn't delete this yet to check if it helps, but I think FTP Simple MUST update this.

What do you think?

**Comment by timurarefev** [comment]
I have the same problem. And also I need a function which would be refresh only selected directory.

**Comment by Alex360hd** [comment]
+1, refreshing directory would be a good option to refresh directory when you deal with your files with another program at the same time.

**Comment by renanmedina** [comment]
Yeah. I had the same issue. I created the file directly on the FTP and i'm not able to "refresh" a folder to get the changes from there. ;/

**Comment by josemarcomg** [comment]
I discovered a workaround. You have to eliminate the directory from workspace in file menu, and then connect to it again. Hope this helps for further references. Greetings :D

**Comment by doghousedev** [comment]
Thanks! This worked for me.  Not ideal, but it works.
